docs/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
-DustExtinction = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+DustExtinction = "fb44c06c-c62f-5397-83f5-69249e0a3c8e"
```

Comment:
I should check documentation again, but I seem to remember that we shouldn't depend on the package itself here, because we want to use the local package that is in the local environment

Reply:
You're right, it shouldn't be there!